app/src/main/res/values/strings.xml

```diff
@@ -26,154 +26,154 @@
     <string name="title_remember">Remember</string>
     <string name="title_export">Export</string>
     <string name="title_import">Import</string>
-    <string name="title_view_only_mode">View-only Mode</string>
-    <string name="title_hide_remote_cursor">Hide Remote Cursor</string>
-    <string name="title_key_compat_mode">Keyboard Compatibility Mode</string>
-    <string name="title_use_ssh_tunnel">Use SSH Tunnel</string>
-    <string name="title_unknown_ssh_host">Unknown SSH Host</string>
-    <string name="title_ssh_host_key_changed">Host Key changed!</string>
+    <string name="title_view_only_mode">View-only</string>
+    <string name="title_hide_remote_cursor">Hide remote cursor</string>
+    <string name="title_key_compat_mode">Keyboard compatibility mode</string>
+    <string name="title_use_ssh_tunnel">SSH tunnel</string>
+    <string name="title_unknown_ssh_host">Unknown SSH host</string>
+    <string name="title_ssh_host_key_changed">The host key changed!</string>
     <string name="title_continue">Continue</string>
-    <string name="title_delete_servers_before_import">Delete current Servers before Import</string>
+    <string name="title_delete_servers_before_import">Remove all servers before importing</string>
     <string name="title_password">Password</string>
-    <string name="title_use_repeater">Use Repeater</string>
-    <string name="title_private_key">Private Key</string>
+    <string name="title_use_repeater">Repeater</string>
+    <string name="title_private_key">Private key</string>
     <string name="title_change">Change</string>
     <string name="title_about">About</string>
-    <string name="title_license">Licence</string>
-    <string name="title_open_source_libraries">Open Source Libraries</string>
-    <string name="title_report_bug">Report a Bug</string>
-    <string name="title_github_repo">Github Repo</string>
+    <string name="title_license">GPLv3+ licence</string>
+    <string name="title_open_source_libraries">Libre libraries</string>
+    <string name="title_report_bug">Report</string>
```

Comment:
```suggestion
    <string name="title_report_bug">Report a bug</string>
```
`Report` is too short, and doesn't provide enough context about the action. We could use `Report an issue` here, as `Bug` may not be a familiar term to non-programmers.

Reply:
I was originally trying to see if I could do without "word 1 word 2" to avoid the title case issue.